tests/integration/sync/sync_integ_base.py

```diff
@@ -1,19 +1,143 @@
 import os
+
+import logging
 import json
-from unittest import TestCase
+import shutil
+import time
+import uuid
+from pathlib import Path
+
+import boto3
+from botocore.exceptions import ClientError
+from botocore.config import Config
+
+from samcli.lib.bootstrap.bootstrap import SAM_CLI_STACK_NAME
+from tests.integration.buildcmd.build_integ_base import BuildIntegBase
+from tests.integration.package.package_integ_base import PackageIntegBase
 
+CFN_SLEEP = 3
+CFN_PYTHON_VERSION_SUFFIX = os.environ.get("PYTHON_VERSION", "0.0.0").replace(".", "-")
+RETRY_ATTEMPTS = 20
+RETRY_WAIT = 1
 
-class SyncIntegBase(TestCase):
+LOG = logging.getLogger(__name__)
+
+
+class SyncIntegBase(BuildIntegBase, PackageIntegBase):
     @classmethod
     def setUpClass(cls):
-        pass
+        PackageIntegBase.setUpClass()
+
+        cls.test_data_path = Path(__file__).resolve().parents[1].joinpath("testdata", "sync")
 
     def setUp(self):
+        self.cfn_client = boto3.client("cloudformation")
+        self.ecr_client = boto3.client("ecr")
+        self.lambda_client = boto3.client("lambda")
+        self.api_client = boto3.client("apigateway")
+        self.sfn_client = boto3.client("stepfunctions")
+        self.sns_arn = os.environ.get("AWS_SNS")
+        self.stacks = []
+        time.sleep(CFN_SLEEP)
+        self.s3_prefix = uuid.uuid4().hex
         super().setUp()
 
     def tearDown(self):
+        shutil.rmtree(os.path.join(os.getcwd(), ".aws-sam", "build"), ignore_errors=True)
```

Comment:
Do we need to remove `build.toml` file as well, which is at `.aws-sam/build.toml` location?

Reply:
Let's remove the whole .aws-sam folder instead, it should be nothing in there that's not removable right?